ReactNativeClient/lib/KeymapUtils.ts

```diff
@@ -0,0 +1,43 @@
+const { shim } = require('lib/shim');
+
+export const DOMtoElectronAccelerator = (e: React.KeyboardEvent<HTMLDivElement>) => {
+	const parts = [];
+	const { key, ctrlKey, metaKey, altKey, shiftKey } = e;
+
+	// First, the modifiers
+	if (ctrlKey) parts.push('Ctrl');
+	if (shim.isMac()) {
+		if (altKey) parts.push('Option');
+		if (shiftKey) parts.push('Shift');
+		if (metaKey) parts.push('Cmd');
+	} else {
+		if (altKey) parts.push('Alt');
+		if (shiftKey) parts.push('Shift');
+	}
+
+	// Finally, the key
+	const _key = DOMtoElectronKey(key);
+	if (_key) parts.push(_key);
+
+	return parts.join('+');
+};
+
+export const DOMtoElectronKey = (key: string) => {
+	if (/^([A-Z0-9]|F1*[1-9]|F10|F2[0-4]|[`~!@#$%^&*()\-_=[\]\\{}|;':",./<>?]|Enter|Tab|Backspace|Delete|Insert|Home|End|PageUp|PageDown|Escape|MediaStop|MediaPlayPause|PrintScreen])$/.test(key)) return key;
+	if (/^([a-z])$/.test(key)) return key.toUpperCase();
+	if (/^Arrow(Up|Down|Left|Right)|Audio(VolumeUp|VolumeDown|VolumeMute)$/.test(key)) return key.slice(5);
```

Comment:
I think I've seen this code in KeyMapService. It should only be in one place. In fact I'm not entirely sure you should have a utils class when you already have KeyMapService. It's going to cause duplicate functionalities and code.

Reply:
It's a bit different. But I refactored the function so that it can use the same RegEx you've mentioned. 

> In fact I'm not entirely sure you should have a utils class when you already have KeyMapService.

I see. I'll be adding these two functions to KeymapService as static functions once that PR gets merged.